config/jobs/kubernetes/sig-release/release-branch-jobs/1.16.yaml

```diff
@@ -214,6 +215,9 @@ periodics:
       - --gcp-project-type=scalability-project
       - --gcp-zone=us-east1-b
       - --provider=gce
+      # 1.16 releases and older need this env var for proper logexporter usage.
```

Comment:
Could you extract 1.16 fixes to a separate commit or PR?

Reply:
we are not testing 1.16 jobs anymore, right? this release version is deprecated

Reply:
we are still testing 1.16

Reply:
Ok thanks

Reply:
>Could you extract 1.16 fixes to a separate commit or PR?

We couldn't. After offline discussion decided to make the comment providing the context a bit more verbose.

Reply:
Thanks!